src/java.base/share/classes/sun/security/provider/SunEntries.java

```diff
@@ -100,10 +100,40 @@
 public final class SunEntries {
 
     /* The property 'jdk.nativeDigest' is used to control enablement of the native
-     * digest implementation.
+     * digest implementations.
      */
     private static final boolean useNativeDigest = NativeCrypto.isAlgorithmEnabled("jdk.nativeDigest", "MessageDigest");
 
+    /* The property 'jdk.nativeMD5' is used to control enablement of the native
+     * MD5 implementations.
```

Comment:
This (and perhaps all the new fields) should say "implementation" instead of "implementations".

Reply:
Fixed.